site2/docs/deploy-bare-metal.md

```diff
@@ -7,9 +7,9 @@ sidebar_label: Bare metal
 
 > ### Tips
 >
-> 1. Single-cluster Pulsar installations should be sufficient for all but the most ambitious use cases. If you are interested in experimenting with
-> Pulsar or using Pulsar in a startup or on a single team, it is simplest to opt for a single cluster. If you do need to run a multi-cluster Pulsar instance,
-> see the guide [here](deploy-bare-metal-multi-cluster.md).
+> 1. Single-cluster Pulsar installation should be sufficient for most use cases. If you are interested in experimenting with
+> Pulsar or using Pulsar in a startup or on a single team, it is simplest to opt for a single cluster. If you need to run a multi-cluster Pulsar instance,
+> see the [guide](deploy-bare-metal-multi-cluster.md).
```

Comment:
```suggestion
> 1. If you are interested in testing or using Pulsar, the easiest way is to run it on a single cluster. A single-cluster Pulsar instance supports most use cases. You can also refer to this [guide](deploy-bare-metal-multi-cluster.md) to run a multi-cluster Pulsar instance.
```

The two sentences should be in one line. It's unnecessary to put them in three lines. The changes are for your reference. You can modify the sentence by yourself.